cluster/addons/fluentd-gcp/fluentd-gcp-image/Dockerfile

```diff
@@ -20,43 +20,39 @@
 # scope and that the Logging API has been enabled for the project
 # in the Google Developer Console.
 
-FROM gcr.io/google_containers/ubuntu-slim:0.6
-
-# Disable prompts from apt
-ENV DEBIAN_FRONTEND noninteractive
-
-# Install build tools
-RUN apt-get -qq update && \
-    apt-get install -y -qq curl ca-certificates gcc make bash sudo && \
-    apt-get install -y -qq --reinstall lsb-base lsb-release && \
-    # Install logging agent and required gems
-    /usr/bin/curl -sSL https://toolbelt.treasuredata.com/sh/install-ubuntu-xenial-td-agent2.sh | sh && \
-    sed -i -e "s/USER=td-agent/USER=root/" -e "s/GROUP=td-agent/GROUP=root/" /etc/init.d/td-agent && \
-    td-agent-gem install --no-document fluent-plugin-record-reformer -v 0.8.2 && \
-    td-agent-gem install --no-document fluent-plugin-systemd -v 0.0.5 && \
-    td-agent-gem install --no-document fluent-plugin-google-cloud -v 0.5.6 && \
-    td-agent-gem install --no-document fluent-plugin-detect-exceptions -v 0.0.4 && \
-    td-agent-gem install --no-document fluent-plugin-prometheus -v 0.2.1 && \
-    # Remove build tools
-    apt-get remove -y -qq gcc make && \
-    apt-get autoremove -y -qq && \
-    apt-get clean -qq && \
-    # Remove unnecessary files
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
-        /opt/td-agent/embedded/share/doc \
-        /opt/td-agent/embedded/share/gtk-doc \
-        /opt/td-agent/embedded/lib/postgresql \
-        /opt/td-agent/embedded/bin/postgres \
-        /opt/td-agent/embedded/share/postgresql \
-        /etc/td-agent/td-agent.conf
+FROM gcr.io/google-containers/debian-base-amd64:0.1
```

Comment:
SGTM, but we should probably formalize this, since we don't have a clear policy on this yet. (e.g `fluentd-gcp:1.28` doesn't point to `1.28.2` right now, but it maybe would in the future?)